src/main/java/com/epam/code/mie/library/controllers/LibraryController.java

```diff
@@ -31,4 +31,11 @@ public ResponseEntity<BookDto> getBookByName(@RequestParam String name) {
         .map(ResponseEntity::ok)
         .orElse(ResponseEntity.notFound().build());
   }
+
+  @Operation(summary = "Add a new book")
+  @PostMapping("/books")
+  public ResponseEntity<BookDto> addBook(@Valid @RequestBody BookDto bookDto) {
```

Comment:
I want to implement here extension of spring validation, here's the steps:
1. Create annotation @AuthorExists and mark AuthorDto field
2. Create class that will implement ConstraintValidator to support this annotation
3. Call AuthorsRepository to check if author with such name, secondName and lastName is present. If present validation is passed, otherwise - failed